File: 137-single-number-ii/single-number-ii.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        for (int i = 0; i < nums.size(); i++) {
            int count = 0;
            bool check = false;
            for (int j = 0; j < nums.size(); j++) {
                if (nums[j] == nums[i])
                    count++;
                if (count > 1) {
                    check = true;
                    break;
                }
            }
            if (!(check)) {
                return nums[i];
            }
        }
        return -1;
    }
};

File: 137-single-number-ii/single-number-ii.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNumber = function(nums) {
    let temp={};
    for(let i=0; i<nums.length; i++){
        if(temp[nums[i]])temp[nums[i]]++;
        else temp[nums[i]]=1;
    }
    for(let key in temp){
        if(temp[key]==1)return Number(key);
    }
};